src/components/DateTimeField/index.jsx

```diff
@@ -74,6 +74,10 @@ class DateTimeField extends React.Component {
       type,
       pattern,
       placeholder,
+      meta: {
+        touched,
```

Comment:
why is touched needed here? Why cant the decision be done with error variable alone?

Reply:
this is to prevent showing errors unnecessarily. Consider a form has just rendered with a required Datefield. Since the field is empty,  the error prop will be true. However we do not wish to show it as an error, as the user has yet to interact with the field. To display errors properly, we need to keep track of whether the user has interacted with the field or not. For this we need the touched prop which will become active only if the user has touched the field e.g by clicking on it.

Reply:
Ok. Is the prop coming from the react parent component?

Reply:
ummm yeah... we are rendering it inside redux-form so we don't pass the meta prop explicitly but redux-form automagically does that for us